core/src/main/java/org/mule/runtime/core/api/extension/MuleExtensionModelDeclarer.java

```diff
@@ -191,6 +191,9 @@ private void declareIdempotentValidator(ExtensionDeclarer extensionDeclarer, Cla
         .withExpressionSupport(NOT_SUPPORTED)
         .describedAs("Defines the prefix of the object store names. This will only be used for the internally built object store.");
 
+    validator.withOptionalComponent("privateObjectStore")
+        .describedAs("Defines a private object store to use instead of the global one for exclusive access");
```

Comment:
In this PR you modify both the redelivery policy and the idempotent validator. However, you're only modifying one element in this extension model